placement-rules-in-sql.md

```diff
@@ -0,0 +1,154 @@
+---
+title: Placement Rules in SQL
+summary: Learn how to schedule placement of tables and partitions.
+---
+
+# Placement Rules in SQL
+
+> **Warning:**
+>
+> Placement Rules in SQL is an experimental feature. The syntax might change before its GA, and there might also be bugs.
+>
+> If you understand the risks, you can enable this experiment feature by executing `SET GLOBAL tidb_enable_alter_placement = 1;`.
+
+Placement Rules allow you to configure where data will be stored in a TiKV cluster. This is useful for scenarios including optimizing a high availability strategy, ensuring that local copies of data will be available for local stale reads, and adhering to compliance requirements.
```

Comment:
Could you explain what is Placement Rules? What is the relationship between the existing [Placement Rules implemented via PD](https://docs.pingcap.com/tidb/dev/configure-placement-rules)?

Maybe some text in https://docs.pingcap.com/tidb/dev/configure-placement-rules explains in more detail. Is is possible to borrow or copy some of the text from that document? For example:

It is a replica rule system that guides PD to generate corresponding schedules for different types of data. By combining different scheduling rules, you can finely control the attributes of any continuous data range, such as the number of replicas, the storage location, the host type, whether to participate in Raft election, and whether to act as the Raft leader.

Reply:
I actually think it does state what they do this in this sentence:

> Placement Rules allow you to configure where data will be stored in a TiKV cluster. This is useful for scenarios including optimizing a high availability strategy, ensuring that local copies of data will be available for local stale reads, and adhering to compliance requirements.

The only part that it misses is the relationship to the previous implementation which is it is _configurable in SQL._

This part is too complicated for users:

> It is a replica rule system that guides PD to generate corresponding schedules for different types of data. By combining different scheduling rules, you can finely control the attributes of any continuous data range, such as the number of replicas, the storage location, the host type, whether to participate in Raft election, and whether to act as the Raft leader.

It is actually not our goal to have multiple different schedules combined, since that increases the risk of misconfiguration. This is why we created the concept of a `PLACEMENT POLICY`, so you can state one policy and apply it everywhere.

Reply:
Thanks for explaning. LGTM.